src/odemis/acq/stream/_base.py

```diff
@@ -42,6 +42,28 @@
 # use hardcode list of polarization positions necessary for polarimetry analysis
 POL_POSITIONS = (MD_POL_HORIZONTAL, MD_POL_VERTICAL, MD_POL_POSDIAG,
                  MD_POL_NEGDIAG, MD_POL_RHC, MD_POL_LHC)
+# user-friendly look-up dict for display in legend
+POL_POSITIONS_DICT = {MD_POL_HORIZONTAL: "horizontal", MD_POL_VERTICAL: "vertical",
```

Comment:
DICT is just explaining the type. Usually it's possible to describe more the purpose of the variable. For instance, here, `POL_POSITIONS_2_NAME` or `POL_POSITIONS_2_DISPLAY`.